components/centraldashboard/app/server.ts

```diff
@@ -33,6 +39,7 @@ const {
   REGISTRATION_FLOW = "true",
   PROMETHEUS_URL = undefined,
   METRICS_DASHBOARD = undefined,
+  METRICS_PORT = 9100,
```

Comment:
Should we expose this also in the [Dockerfile](https://github.com/rgildein/kubeflow/blob/94995d6a519183fc19a25ba36a5f23cd6b78f681/components/centraldashboard/Dockerfile#L30-L31)?

Reply:
and probably manifests too?
https://github.com/kubeflow/kubeflow/blob/54201e3d28e4673d465e47e464e86aa698ca0dfa/components/centraldashboard/manifests/base/deployment.yaml#L29-L31
https://github.com/kubeflow/kubeflow/blob/master/components/centraldashboard/manifests/base/service.yaml#L9-L11

Reply:
What do you say if we change this to be only `METRICS`, which will enable or disable the metrics and port will be always 9100? I don't think there is much need to change the port, but we can use both `METRICS` and `METRICS PORT`. WDYT?